release/release_tests.yaml

```diff
@@ -2117,7 +2117,53 @@
 
   cluster:
     cluster_env: app_config.yaml
-    cluster_compute: 8gpus_64cpus.yaml
+    cluster_compute: 8gpus_96cpus.yaml
+
+  run:
+    timeout: 14400
+    script: python learning_tests/run.py --yaml-sub-dir=a-e --test-name=rllib_learning_tests_a_to_e
+    type: sdk_command
+    file_manager: job
+
+  alert: default
+
+- name: rllib_learning_tests_f_to_z
+  group: RLlib tests
+  working_dir: rllib_tests
+
+  legacy:
+    test_name: learning_tests
+    test_suite: rllib_tests
+
+  frequency: nightly
+  team: ml
+
+  cluster:
+    cluster_env: app_config.yaml
+    cluster_compute: 8gpus_96cpus.yaml
+
+  run:
+    timeout: 14400
+    script: python learning_tests/run.py --yaml-sub-dir=a-e --test-name=rllib_learning_tests_f_to_z
+    type: sdk_command
+    file_manager: job
+
+  alert: default
+
+- name: rllib_learning_tests_e_to_z
```

Comment:
is this how the tests are actually split?

Reply:
look in the yaml dir, files are split into sub-dirs. if you have better ideas, let me know.

Reply:
no this is good as is just was checking.